json-rpc/types/src/views.rs

```diff
@@ -286,7 +286,7 @@ pub enum EventDataView {
         role_id: u64,
     },
     #[serde(rename = "unknown")]
-    Unknown {},
+    Unknown { bytes: Option<BytesView> },
```

Comment:
Can this ever be `None` (and if so, when)?

Reply:
The views are used by client sdk too, thus, when the client connects to a fullnode that is older version, you get none.
@bmwill do you want to decouple client sdk views with server views? It seems confusing to use client sdk views as server views.

Reply:
I would really like to try to not use different struct definitions for client and server, because it will end up with just a lot of duplicate code if we do.

Reply:
Here's an idea, really we just need to have a variant that we can use as a default for clients that don't know anything and if, before this change there weren't any way of getting unknown events (between 1.1 and 1.2 really), then we can maybe do the following:
```
    #[serde(rename = "unknown")]
    Unknown { bytes: BytesView },

    #[serde(other)]
    UnknownToClient,
```